credentialproviderpackage/charts/credential-provider-package/templates/daemonset.yaml

```diff
@@ -50,15 +50,15 @@ spec:
             - name: OS_TYPE
               value: {{ $os }}
             - name: AWS_PROFILE
-              value: {{.Values.application.profile}}
+              value: {{(index .Values.credential 0).profile}}
             - name: MATCH_IMAGES
-              value: '{{ join "," .Values.application.matchImages }}'
+              value: '{{ join "," (index .Values.credential 0).matchImages }}'
             - name: DEFAULT_CACHE_DURATION
-              value: {{.Values.application.defaultCacheDuration}}
+              value: {{(index .Values.credential 0).defaultCacheDuration}}
       volumes:
         - name: aws-creds
           secret:
-            secretName: {{.Values.application.secretName}}
+            secretName: {{(index .Values.credential 0).secretName}}
```

Comment:
Add a comment mentioning we only support 1 secret for now and it's the aws-secret.